app-config-utils/src/index.ts

```diff
@@ -26,7 +26,7 @@ export function isPrimitive(obj: Json): obj is JsonPrimitive {
 }
 
 export function generateModuleText(
-  fullConfig: Json,
+  fullConfig: Json | 'no-config',
```

Comment:
Is there a reason we are using arbitrary `'no-config'` string over `undefined`? Is it because `undefined` is included in `Json`?

Reply:
This was just the first thing that came to mind, since `undefined` is a little bit ambiguous (would mean consumers might accidentally not pass an argument)

Reply:
I'd argue that not passing any argument _does_ show intention to not bundle any config (what else should the consumer expect given the context of this function?). I won't die on a hill for this one though.

Reply:
Right, I agree sorry, was more talking about accidentally not passing the argument, but if it's `| undefined` typescript should protect against that